public/docs/content/examples-statusIndicator-Dropdown.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "type": "paragraph",
+    "children": [
+      {
+        "text": "StatusIndicators are commonly used in the dropdown menu."
```

Comment:
Let's change to - "Here the example of using StatusIndicator in the dropdown menu."

Reply:
done